src/Symfony/Component/DependencyInjection/Tests/Fixtures/xml/legacy-services9.xml

```diff
@@ -33,6 +33,6 @@
       <configurator class="%baz_class%" method="configureStatic1"/>
     </service>
     <service id="factory_service" class="Bar" factory-method="getInstance" factory-service="foo.baz"/>
-    <service id="foo_bar" class="%foo_class%" scope="prototype"/>
+    <service id="foo_bar" class="%foo_class%" shared="false" scope="prototype"/>
```

Comment:
Yes, this is needed. see [here](https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/DependencyInjection/Dumper/XmlDumper.php#L129-L131) and [here](https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/DependencyInjection/Dumper/YamlDumper.php#L131-L133). All services of `prototype` scope are non-shared by default.